addons/mrp/static/src/widgets/timer.js

```diff
@@ -24,55 +41,50 @@ export class MrpTimer extends Component {
                 ? this.props.ongoing
                 : this.props.record.data.is_user_working;
 
-        onWillStart(() => this._runTimer());
+        onWillStart(async () => {
+            const addWorkingDuration = 
+                this.props.ongoing !== undefined
+                    ? false
+                    : this.props.record.data.state == "progress";
+            if(addWorkingDuration) {
```

Comment:
```suggestion
            if (this.props.ongoing ===undefined && this.props.record.data.state == "progress") {
```
as you don't need the value of addWorkingDuration later